src/components/QuestionsGrid/QuestionsGrid.svelte

```diff
@@ -0,0 +1,28 @@
+<script lang="ts">
+import QuestionCard from '../QuestionCard/QuestionCard.svelte'
+
+export let questions = [
+  {
+    level: 1,
+    title: 'Title',
+    description: 'Description'
+  }
+]
```

Comment:
please leave this as an empty array. By putting the keyword "export" this will get overwritten by what is passed in.